skll/utilities/filter_features.py

```diff
@@ -104,10 +111,25 @@ def main(argv=None):
                       'file.  You specified: {}').format(output_extension))
         sys.exit(1)
 
+    if input_extension == '.csv' or input_extension == '.tsv':
+        replace_blanks_with = args.replace_blanks_with
+        ignore_blanks = args.ignore_blanks
+        if ignore_blanks and replace_blanks_with is not None:
+            raise ValueError("You cannot pass a value for 'replace_blanks_with' "
+                             "and set 'ignore_blanks' to True.")
```

Comment:
How about the same message as before: "You cannot both drop blanks and replace them. 'replace_blanks_with' can only have a value when 'drop_blanks' is `False`."